src/styles/global.ts

```diff
@@ -20,5 +19,6 @@ export const global = css`
   #root {
     width: 100%;
     height: 100%;
+    background-color: #fbfbff;
```

Comment:
```suggestion
    background-color: ${COLOR.background};
```
색상은 color.ts에 선언한 것을 사용해서 이렇게 작성하는 것이 좋을 것 같아용!